OpenVPN/openvpn/_meta/smart-descriptions.json

```
@@ -60,6 +60,14 @@
       }
     ]
   },
+  {
+    "value": "Authentication failure for user {user.name}",
+    "conditions": [
+      { "field": "event.category", "value": "authentication" },
+      { "field": "event.type", "value": "end" },
```

I suggest improving the condition
```suggestion
      { "field": "event.type", "value": "start" },
      { "field": "event.outcome", "value": "failure" },
```